flint.ui/src/store/modules/rothc.js

```diff
@@ -241,9 +241,15 @@ export default {
     RothC_stepLenInYears: [],
     initialValues: [],
     plantCM: [],
-    atmosphere: []
+    atmosphere: [],
+    firstRun: true
   },
 
+  mutations: { 
+    setRunStatus(state, value) { 
+      state.firstRun = value 
+    },
+
   mutations: {
     setNew_rothc_startDate(state, newValue) {
       this.state.rothc.config.LocalDomain.start_date = newValue
```

Comment:
@Bhawna1203 The build currently fails. It's probably because you've defined a different `mutations` section. Please add your mutation under the existing mutations section starting from line 253.

Reply:
okk @Crystalsage sir done :)